packages/jest-docblock/src/__tests__/index.test.ts

```diff
@@ -58,16 +58,11 @@ describe('docblock', () => {
   });
 
   it('returns extract and parsedocblock', () => {
-    const code =
-      '/** @provides module-name */' +
-      EOL +
-      '' +
-      '' +
-      EOL +
-      '' +
-      '.dummy {}' +
-      EOL +
-      '';
+    const code = String(
```

Comment:
remove all the `String` stuff and use template strings instead

Reply:
Sure

Reply:
I'll fix that on master

Reply:
sure